app/src/main/res/layout/activity_speakers.xml

```diff
@@ -1,6 +1,7 @@
 <android.support.design.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/speaker"
```

Comment:
please follow naming conventions. `speaker` does not tell me what is this id used for. is it a `TextView` ? Then it should be `txtSpeakerName` in case of speaker name, in this case a better name would be `speakerRootLayout` or `speakerFrameLayout` or `speakerOuterLayout` etc. I hope you get the idea.